tests/testthat/test-interact.R

```diff
@@ -79,25 +79,53 @@ with_test_authentication({
                 missing = FALSE
             )
         )
-        expect_json_equivalent(
-            test_cases[[10]],
-            list(
-                expression = ds$q1 == "Not Asked" & ds$country == "Australia",
-                name = "Not Asked:Australia",
-                missing = TRUE
+        tryCatch(
+            expect_json_equivalent(
+                test_cases[[11]],
+                list(
+                    expression = ds$q1 == "Not Asked" & ds$country == "Australia",
+                    name = "Not Asked:Australia",
+                    missing = TRUE
+                )
+            ),
+            error = function(e) expect_json_equivalent(
+                test_cases[[10]],
+                list(
+                    expression = ds$q1 == "Not Asked" & ds$country == "Australia",
+                    name = "Not Asked:Australia",
+                    missing = TRUE
+                )
```

Comment:
This isn't absolutely necessary since this works, but a slightly more straight-forward way to do this would be `if`/`else` conditioned on `length(test_cases) == 11`. It reads a bit more of the intention there.